extension/lib/magento/customer/getItems.js

```diff
@@ -13,8 +13,13 @@ module.exports = async (context, input) => {
   const wishlistItems = await request.send(wishlistItemsEndpointUrl, 'Request to Magento: getFavorites', 'GET')
   context.storage.user.map.del('wishlistItemIdMapping')
   const productIds = wishlistItems.map(item => {
-    context.storage.user.map.setItem('wishlistItemIdMapping', item.product_id, item.wishlist_item_id)
-    return item.product_id
+    const { child_ids: childIds, type, product_id: productItemId } = item
+    let productId = productItemId
+    if (type === 'configurable' && childIds && childIds.length === 1) {
```

Comment:
we technically have tests for this in the cloud plugin, I don't think we should be this defensive. That is, if magento endpoint does return multiple id's for a configurable product OR no child ids, this implementation would only hide the problem. Just trust the endpoint imho. @Rekhyt what ya think?

Reply:
Done